odoo/addons/base/tests/test_configmanager.py

```diff
@@ -425,6 +425,10 @@ def test_04_odoo16_config_file(self):
                 'smtp_ssl_certificate_filename',
                 'smtp_ssl_private_key_filename', 'smtp_user'
             )
+        ] + [
+            ("WARNING:odoo.tools.config:Since 19.0, option "
+             f'"without_demo=False" in the config file at {config_path}'
+             ' should be replaced by "with_demo=True"'),
```

Comment:
I wonder if we need this warning, it is not like we are going to remove the option in the future

Reply:
I think it is good and avoids confusion: `odoo-bin --with-demo --without-demo` - euh?
We should remove the option in a few years.